docs/uncertainty/index.rst

```diff
@@ -339,9 +339,10 @@ example above, but with 200x fewer samples:
   >>> plt.scatter(n1.distribution, n2.distribution, s=5, lw=0) # doctest: +SKIP
   >>> plt.xlim(-4, 4) # doctest: +SKIP
   >>> plt.ylim(-4, 4) # doctest: +SKIP
-  >>> np.cov(n1.distribution, n2.distribution) # doctest: +FLOAT_CMP
-  array([[1.04667972, 0.19391617],
-         [0.19391617, 1.50899902]])
+
+>>> np.cov(n1.distribution, n2.distribution) # doctest: +FLOAT_CMP

```

Comment:
Huh!?!? Does the indentation really matter here? This is so very weird!

Reply:
Yes, because when it is inside `.. plot::`, [matplotlib's plot directive](https://matplotlib.org/3.1.3/devel/plot_directive.html) takes over and somehow it stopped recognizing `FLOAT_CMP` defined by `pytest-doctestplus`. Getting this outside of the plot directive is the quickest fix, unless someone else want to hunt down why suddenly this is happening.

Reply:
Wow...  But to make it look better, how about adding a sentence above, like
```
The impression one gets from the plot is confirmed from the covariance matrix::

   >>> np.cov(n1.distribution, n2.distribution) # doctest: +FLOAT_CMP
```

Reply:
Good idea. Done. Thanks!